config.libsonnet

```diff
@@ -12,7 +12,7 @@
     clusterVersionAlertTime: '10m',
     clusterUtilizationAlertTime: '5m',
     monQuorumAlertTime: '15m',
-    monQuorumLeaderChangesAlertTime: '30m',
+    monQuorumLeaderChangesAlertTime: '0s',
```

Comment:
I am assuming this is `0s` for getting instant alerts.
But, what about cases where cluster is getting initialized and mon experiences frequent changes in leader for some reason?
I would say this should be set to something `>= 5m`.

Reply:
You are right, I didn't think of cluster creating. Changed to 5m. Please take a look